internal/checker/terraform/rules/azure/azurerm_storage_account_storage_logging_enabled.rego

```diff
@@ -0,0 +1,54 @@
+# METADATA
+# title: "Confirm that Storage logging is activated including read, write, and delete requests for the Queue service"
+# description: "This check confirms that Storage logging is configured to capture activity within the Queue service, specifically for read, write, and delete requests. Activating this logging provides valuable insights into Queue service operations for monitoring and auditing purposes."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account
+# custom:
+#   id: CB_TFAZR_027
+#   severity: MEDIUM
+package lib.terraform.CB_TFAZR_027
+
+
+isvalid(block){
+	block.Type == "resource"
+    block.Labels[_] == "azurerm_storage_account"
+}
+
+resource[resource] {
+    block := pass[_]
+	resource := concat(".", block.Labels)
+} 
+
+resource[resource] { 
+    block := fail[_]
+	resource := concat(".", block.Labels)
+} 
+
```

Comment:
check if creating another pass block is necessary

check attribute "account_kind" != Storage and StorageV2